driver/config/config.go

```diff
@@ -527,10 +539,37 @@ func (p *Config) SecretsSession() [][]byte {
 	return result
 }
 
+func (p *Config) SecretsAES() (secret string) {
+	secret = p.p.String(ViperKeySecretsAES)
+	if len(secret) == 0 {
+		secret = string(p.SecretsDefault()[0][:32])
```

Comment:
This will panic if the secret is not 32 chars long

Reply:
use the key generator from cryptopasta